packages/babel-plugin-transform-parameters/test/fixtures/assumption-noNewArrows-false/default/output.js

```diff
@@ -0,0 +1,7 @@
+var _this = this;
```

Comment:
This is another bug fixed by the new `if (!path.isFunction()) return;` check in the plugin.